tests/robot-tests/tests/public_api/public_api_dataset_preview_token.robot

```diff
@@ -0,0 +1,284 @@
+*** Settings ***
+Library             ../libs/admin_api.py
+Resource            ../libs/admin-common.robot
+Resource            ../libs/admin/manage-content-common.robot
+Resource            ../libs/public-api-common.robot
+Resource            ../libs/public-common.robot
+
+Force Tags          Admin    Local    Dev    AltersData
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+
+
+
+*** Variables ***
+${PUBLICATION_NAME}=    UI tests - public api preview token %{RUN_IDENTIFIER}
+${RELEASE_NAME}=        Financial year 3000-01
```

Comment:
I would suggest using a different release name so that we do not run into the validation error 'publicationSlugNotUnique' when other also run in the Public_api suite.